pkgs/by-name/mu/musl/package.nix

```diff
@@ -0,0 +1,177 @@
+{
+  lib,
+  callPackage,
+  fetchurl,
+  linuxHeaders,
+  stdenv,
+  # Boolean flags
+  hasLinuxHeaders ? true,
+  useBSDCompatHeaders ? true,
+}:
+let
+  _sources = callPackage ./sources.nix { };
```

Comment:
Wouldn't it be better to inherit parameters here?

Reply:
Well, `_source` is because I wanted that `source` could be passed as argument; however it can clash with users of third-party overlays that define `source`.

It happened when I did similar thing at nv-codec-headers:

Reply:
`sourceInputs`?

Reply:
Problem is, anyone on the planet needs to agree with some convention...